From b10794eff12086eef08e1307f7c43b0c84092415 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Tue, 24 Jul 2018 14:04:34 +0800
Subject: [PATCH 138/151] dt-bindings: display: sunxi: sun8i-dw-hdmi: add
 ddc-en gpio

Some boards have a GPIO that controls the level shifting transistor of
HDMI DDC pins.

Add a ddc-en-gpios property to sun8i-dw-hdmi , which refer to such GPIOs.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 Documentation/devicetree/bindings/display/sunxi/sun4i-drm.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/sunxi/sun4i-drm.txt b/Documentation/devicetree/bindings/display/sunxi/sun4i-drm.txt
index 11e82235048e..e45372cb5395 100644
--- a/Documentation/devicetree/bindings/display/sunxi/sun4i-drm.txt
+++ b/Documentation/devicetree/bindings/display/sunxi/sun4i-drm.txt
@@ -120,6 +120,9 @@ Required properties:
   - resets: phandle to the reset controller driving the PHY
   - reset-names: must be "phy"
 
+Optional properties:
+  - ddc-en-gpios: gate GPIOs for the HDMI DDC pins
+
 H3, A64 and R40 HDMI PHY require additional clocks:
   - pll-0: parent of phy clock
   - pll-1: second possible phy clock parent (A64/R40 only)
-- 
2.18.0

